activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -326,8 +326,11 @@ def _select!(*fields) # :nodoc:
 
     # Add a Common Table Expression (CTE) that you can then reference within another SELECT statement.
     #
-    # Note: CTE's are only supported in MySQL for versions 8.0 and above. You will not be able to
-    # use CTE's with MySQL 5.7.
+    # Note: CTEs are only supported with the following database adaptors:
```

Comment:
:eyes: "adapters" should be better to keep it consistent in the file.

Reply:
Agreed.
English is hard! :grimacing: